Tests/ApplicativeRouterTests/TestRouter.swift

```diff
@@ -34,11 +39,16 @@ let router: Router<Routes> = [
   Routes.iso.postBodyJsonDecodable
     <¢> post %> jsonBody(Episode.self) <%> lit("episodes") %> .int <% end,
 
-  // /path/to/somewhere/cool?ref=:optional_string&active=:bool&t=:int
+  // GET /path/to/somewhere/cool?ref=:optional_string&active=:bool&t=:int
   Routes.iso.simpleQueryParams
     <¢> get %> lit("path") %> lit("to") %> lit("somewhere") %> lit("cool")
     %> queryParam("ref", opt(.string)) <%> queryParam("active", .bool) <%> queryParam("t", .int)
     <% end,
+
+  // GET /subscribe?plan=:int&token=:int
+  Routes.iso.codableQueryParams
+    <¢> get %> lit("subscribe") %> queryParams(SubscribeData.self)
```

Comment:
This type could be inferred, though we also have a custom decoder to worry about, so the inconsistency between a computed `queryParam` vs. default decoder `queryParam()` vs. `queryParam(myDecoder)` is a pain.